x-pack/test/api_integration/apis/uptime/graphql/expect_fixture_eql.ts

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import expect from '@kbn/expect';
+import fs from 'fs';
+import { join } from 'path';
+
+const fixturesDir = join(__dirname, 'fixtures');
+
+export const expectFixtureEql = (data: any, fixtureName: string) => {
```

Comment:
Great! This is along the lines of the solution I was planning to use to address this.